interface/interface.dm

```diff
@@ -1,12 +1,14 @@
 //Please use mob or src (not usr) in these procs. This way they can be called in the same fashion as procs.
-/client/verb/wiki()
+/client/verb/wiki(query as text)
 	set name = "wiki"
-	set desc = "Visit the wiki."
+	set desc = "Type what you want to know about.  This will open the wiki on your web browser."
 	set hidden = 1
 	if(config.wikiurl)
-		if(alert("This will open the wiki in your browser. Are you sure?",,"Yes","No")=="No")

```

Comment:
The replacement is the verb input, it *should* however be "as text|null" to add a cancel, and if null the verb should do nothing.